tests/test_mesh.py

```diff
@@ -264,6 +264,34 @@ def arithmetic_mesh_setup(self):
         flux_data = [1.1, 2.2, 3.3, 4.4]
         self.mesh_2.mesh.tag_set_data(flux_tag, volumes1, flux_data)
 
+    def arithmetic_mesh_vector_setup(self):
+        self.mesh_1_vector = Mesh(structured_coords=[[-1, 0, 1], [-1, 0, 1], [0, 1]],
+                                  structured = True)
+        self.volumes1 = list(self.mesh_1_vector.structured_iterate_hex("xyz"))
+       
+        fracs_vector_tag = self.mesh_1_vector.mesh.tag_get_handle("cell_fracs", 
+                                                          2,
+                                                          types.MB_TYPE_DOUBLE,
+                                                          types.MB_TAG_DENSE,
+                                                          create_if_missing=True)
+
+        fracs_vector_data = [[1.0, 2.0], [1.5, 2.5], [-1.0, -1.5], [-2.0, -2.5]]        
```

Comment:
While this still works for testing, the fracs vector should generally always be positive and sum to 1.  To avoid confusion you could either use a different variable name for this tag.data, or change the data to match that convention.

Reply:
Okay, that makes sense. If I were to just change the name of the tag, should I use a different "real" tag name or perhaps create a "sample" tag name that doesn't typically exist.

Reply:
Some sample tag name that doesn't exist is fine.  PyNE meshes can be used for many different purposes beyond this one, so many things may turn out to be "real" tag names.  I'd go with something that is obviously a tag generated for testing purposes.